daq_macros.py

```diff
@@ -1840,6 +1840,14 @@ def snakeRasterBluesky(rasterReqID, grain=""):
     total_exposure_time = exptimePerCell*totalImages
 
     raster_flyer.configure_detector(file_prefix=rasterFilePrefix, data_directory_name=data_directory_name)
+    if raster_flyer.detector.cam.armed.get() == 1:
+        daq_lib.gui_message('Detector is in armed state from previous collection! Stopping detector, but the user '
+                            'should check the most recent collection to determine if it was successful. Cancelling'
+                            'this collection, retry when ready.')
+        raster_flyer.detector.cam.acquire.put(0)
+        logger.warning("Detector was in the armed state prior to this attempted collection.")
+        return 0
+        #raise RuntimeError("Detector is armed, cannot continue in current state.")
```

Comment:
can you get rid of this last commented-out line?

Reply:
Done.